plugins/TrollManagement/TrollManagementTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * @author David Barbier<david.barbier@vanillaforums.com>
+ * @copyright 2009-2021 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\TrollManagement;
+
+use UserModel;
+use RoleModel;
+use VanillaTests\Models\UserModelTest;
+use VanillaTests\SetupTraitsTrait;
+
+/**
+ * Class TrollManagementTest
+ */
+class TrollManagementTest extends UserModelTest {
```

Comment:
I don't think you want to extend `UserModelTest`. You're going to be running every test in that base class too.

Reply:
Right. Replaced it with SiteTestCase.